### DISCUSSION TITLE
疑问

### DISCUSSION THREAD

**Discussion by kinich-cute** [share121/inter-knot#5031] [委托]
为啥我的绳网左上角界面显示的是未登录呢？

**Comment by LV0Incapableperson** [comment]
"我的" 里面 把公共接口改成私人接口
<img src="https://bbs-static.miyoushe.com/static/2024/09/10/d60393fe879038382e5c7cdcf7ca25fd_7267933421783671773.png" alt="飞霄 打架" width="75" >

**Reply by kinich-cute** [reply]
谢谢，已经搞定了  不过你这表情包是怎么弄的？( ﾟдﾟ )
